Core.DomainServices/Queries/UserQueries/QueryByApiOrStakeHolderAccess.cs

```diff
@@ -0,0 +1,13 @@
+﻿using Core.DomainModel;
+using System.Linq;
+
+namespace Core.DomainServices.Queries.UserQueries
+{
+    public class QueryByApiOrStakeHolderAccess : IDomainQuery<User>
+    {
+        public IQueryable<User> Apply(IQueryable<User> users)
+        {
+            return users.Where(x => x.HasStakeHolderAccess || (x.HasApiAccess.HasValue && x.HasApiAccess.Value));
```

Comment:
`x.HasApiAccess != null && x.HasApiAccess == true`

Jeg har før oplevet at linq to sql compiler bliver lidt pigesur over de der wrapper properties på nullable men altså i virkeligheden kan du reducere udtrykket til `x.HasApiAccess == true`